src/System.CodeDom/tests/project.json

```diff
@@ -31,23 +31,6 @@
     "netstandard1.7": {}
   },
   "supports": {
-    "coreFx.Test.netcoreapp1.1": {
-      "netstandard1.7": [
-        "win7-x86",
-        "win7-x64",
-        "win10-arm64",
-        "osx.10.10-x64",
-        "centos.7-x64",
-        "debian.8-x64",
-        "rhel.7-x64",
-        "ubuntu.14.04-x64",
-        "ubuntu.16.04-x64",
-        "ubuntu.16.10-x64",
-        "fedora.23-x64",
-        "linux-x64",
-        "opensuse.13.2-x64",
-        "opensuse.42.1-x64"
-      ]
-    }
+    "coreFx.Test.netcoreapp1.1": {}
```

Comment:
Can you please also include "coreFx.Test.net463" as it is needed for the other test configuration.